source/tests/unit/server_configuration_parser_tests.cpp

```diff
@@ -613,7 +613,7 @@ TEST_P(ServerConfigurationParserCodeReadinessTests, CodeReadinessConfiguration_P
   EXPECT_EQ(std::get<1>(GetParam()), readiness);
 }
 
-TEST_P(ServerConfigurationParserCodeReadinessTests, CodeReadinessConfiguration_ParseFeatureToggles_FeatureOfCorrespodingReadinessIsEnabled)
+TEST_P(ServerConfigurationParserCodeReadinessTests, CodeReadinessConfiguration_ParseFeatureToggles_FeatureOfCorrespondingReadinessIsEnabled)
```

Comment:
Note: This was a typo fix but also points out that these tests for parsing the "code_readiness" exist (this test and the one directly above) including "RestrictedRelease" that's in the new `localhost_restricted_config.json` file.